include/boost/gil/image_processing/numeric.hpp

```diff
@@ -136,21 +136,26 @@ inline auto generate_gaussian_kernel(std::size_t side_length, double sigma)
         throw std::invalid_argument("kernel dimensions should be odd and equal");
 
     const double denominator = 2 * boost::gil::detail::pi * sigma * sigma;
-    auto middle = side_length / 2;
+    const auto middle = side_length / 2;
     std::vector<T, Allocator> values(side_length * side_length);
+    T sum{0};
     for (std::size_t y = 0; y < side_length; ++y)
     {
-        for (std::size_t x = 0; x < side_length; ++x)
-        {
-            const auto delta_x = middle > x ? middle - x : x - middle;
-            const auto delta_y = middle > y ? middle - y : y - middle;
-            const double power = (delta_x * delta_x +  delta_y * delta_y) / (2 * sigma * sigma);
-            const double nominator = std::exp(-power);
-            const float value = static_cast<float>(nominator / denominator);
-            values[y * side_length + x] = value;
-        }
+      for (std::size_t x = 0; x < side_length; ++x)
+      {
+        const auto delta_x = x - middle;
+        const auto delta_y = y - middle;
+        const auto power = static_cast<double>(delta_x * delta_x +  delta_y * delta_y) / (2 * sigma * sigma);
+        const double nominator = std::exp(-power);
+        const auto value = static_cast<T>(nominator / denominator);
+        values[y * side_length + x] = value;
+        sum += value;
+      }
```

Comment:
Unless this re-formatting is required, I'd suggest to avoid such changes in bugfixing PR-s.

By the way, we have https://github.com/boostorg/gil/blob/develop/example/clang-format/.clang-format

Reply:
Good point, thx for the remark. I guess clion did this for me. Will revert.

Reply:
I wasn't sure if you meant the pure reformatting (e.g. indentation etc) or also the rewrite of `delta_x/y` and `const auto value = static_cast<T>(nominator / denominator);` ... so I just reverted the pure formatting change. 

Btw. If I apply the clang-format file to this file, it change quite a lot actually.

Reply:
I meant the 4 spaces to 2 spaces change.

Yeah, we haven't completed the clang-format application yet, so automated formatting is in a bit of a mess.
Since we're going to do the big reformat sooner or later, then don't bother with the accidental formatting your IDE. 
IOW, ignore my request please.